<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.2.2 (Python 3.13.2 on win32)" generated="2025-08-07T15:14:54.376997" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="c:\Users\thais.regina.santos\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2025-08-07T15:14:54.417822" elapsed="7.295108"/>
</kw>
<status status="PASS" start="2025-08-07T15:14:54.417143" elapsed="7.296047"/>
</test>
<status status="PASS" start="2025-08-07T15:14:54.379133" elapsed="7.335087"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat name="Robot Interactive Console" id="s1" pass="1" fail="0" skip="0">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2025-08-07T15:14:54.367989" level="WARN">Error in file 'c:\Users\thais.regina.santos\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
